src/glossary.rst

```diff
@@ -1298,17 +1298,17 @@ of syntactic transformation rules.
 :dp:`fls_1te2kfi9lt6c`
 See :s:`MacroRulesDeclaration`.
 
-.. _fls_bkkqm3z6ty47:
+.. _fls_GAlaslkO8gLG:
 
-deconstruction type
-^^^^^^^^^^^^^^^^^^^
+deconstructee
+^^^^^^^^^^^^^
 
-:dp:`fls_vsz2v8na8clu`
-The :dt:`deconstruction type` indicates the :t:`type` of the :t:`struct` being
+:dp:`fls_QsvWOdoFWtUO`
+A :t:`deconstructee` indicates the :t:`enum variant` or :t:`type` that is being
```

Comment:
Ah, busted. The proper tag is

:dt:

since it is a definition of a term. This applies to all the new and updated terms.